plugin/download.sh

```diff
@@ -70,6 +70,10 @@ download_assets_for_release() {
     # Node 12
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-12-$tag.sha512" "$dir/heroku-nodejs-plugin-node-12.sha512"
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-12-$tag.tar.gz" "$dir/heroku-nodejs-plugin-node-12.tar.gz"
+
+    # Node 13
```

Comment:
Note: this isn't ran during the build, but rather this is a script to be run in development. These downloaded files should be checked into the git repo

Reply:
Dangit... okay, I need to update these 🙈